### DISCUSSION TITLE
Working Set File Locations

### DISCUSSION THREAD

**Discussion by nschutz** [bernardop/vscode-working-sets#26] [Q&A] [answered]
Hey love the plugin! I'd like to share my working sets between my vs code environments but I'm having trouble tracking down the files that define the working sets.

Does anyone know where I can find them? If there's a way to embed the working sets in the repository that would be really cool too.

Thanks to anyone who sees this and answers!

**Comment by bernardop** [comment, marked as answer]
Does the [saveWorkingSetsInWorkspace](https://github.com/bernardop/vscode-working-sets#extension-settings) accomplish what you're looking for?

**Reply by nschutz** [reply]
Thanks for the answer! Yes it definitely helps, I think because the workings set JSON file stores the absolute path to each file it'll require a find and replace to be useable on another environment, but that's easy enough to do. 

I haven't tried it yet so it may not even require that, thanks again!
